packages/cli-dashboard/src/components/utils/tests/extractCookies.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import extractCookies from '../extractCookies';
+import { tempSinglePageData } from './data.mock';
+
+describe('extractCookies', () => {
+  it('should return an empty object if no cookies are present', () => {
+    expect(extractCookies({}, '')).toEqual({});
+  });
+
+  it('should return an object with the cookies', () => {
+    let isLandingPage = true;
+    const pageUrl = 'https://edition.cnn.com';
+
+    expect(
+      extractCookies(tempSinglePageData.cookieData, pageUrl, isLandingPage)
+    ).toEqual({
+      'https://edition.cnn.com': {
+        'countryCode:.cnn.com:/': {
+          name: 'countryCode',
+          domain: '.cnn.com',
+          path: '/',
+          value: 'IN',
+          sameSite: 'None',
+          expires: 'Session',
+          httpOnly: false,
+          secure: true,
+          isBlocked: false,
+          platform: 'Unknown',
+          category: 'Uncategorized',
+          GDPR: '',
+          description: '',
+          isFirstParty: true,
+          pageUrl,
+          frameUrl: 'https://edition.cnn.com',
+        },
+      },
+      'Unknown frame(s)': {},
+    });
+
+    isLandingPage = false;
+
+    expect(
+      extractCookies(tempSinglePageData.cookieData, pageUrl, isLandingPage)
+    ).toEqual({
+      'https://edition.cnn.com': {
+        ['countryCode:.cnn.com:/' + pageUrl]: {
+          name: 'countryCode',
+          domain: '.cnn.com',
+          path: '/',
+          value: 'IN',
+          sameSite: 'None',
+          expires: 'Session',
+          httpOnly: false,
+          secure: true,
+          isBlocked: false,
+          platform: 'Unknown',
+          category: 'Uncategorized',
+          GDPR: '',
+          description: '',
+          isFirstParty: true,
+          pageUrl,
+          frameUrl: 'https://edition.cnn.com',
+        },
+      },
+      'Unknown frame(s)': {},
```

Comment:
Constant for this particular key has been added [here](https://github.com/GoogleChromeLabs/ps-analysis-tool/blob/eafeffb0f3bebe28e29a34e0681294a60998c3a5/packages/common/src/index.ts#L26).